internal/services/oracle/oracle_exascale_database_virtual_machine_cluster_resource.go

```diff
@@ -0,0 +1,628 @@
+					"total_size_in_gb": {
+						Type:     pluginsdk.TypeInt,
+						Required: true,
+						ForceNew: true,
+					},
+				},
+			},
+		},
+
+		"virtual_network_id": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ForceNew:     true,
+			ValidateFunc: commonids.ValidateVirtualNetworkID,
+		},
+
+		"backup_subnet_cidr": {
+			Type:         pluginsdk.TypeString,
+			Optional:     true,
+			ForceNew:     true,
+			ValidateFunc: validation.IsCIDR,
+		},
+
+		"cluster_name": {
+			Type: pluginsdk.TypeString,
+			// The O+C is specified here because the service will set the virtual machine's name as the default value if not specified.
+			Optional: true,
+			Computed: true,
+			ForceNew: true,
+			ValidateFunc: validation.StringMatch(regexp.MustCompile(`^[a-zA-Z][a-zA-Z0-9-]{0,10}$`),
+				"The Cluster name must begin with an alphabetic character, be no longer than 11 characters, and may contain alphabets, numbers, and hyphens (-).",
+			),
+		},
+
+		"data_collection": {
+			Type: pluginsdk.TypeList,
+			// The O+C is specified here because the service will set a default value to the data_collection if not specified.
+			Optional: true,
+			Computed: true,
+			ForceNew: true,
+			MaxItems: 1,
+			MinItems: 1,
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"diagnostics_events_enabled": {
+						Type:     pluginsdk.TypeBool,
+						Optional: true,
+						ForceNew: true,
+						Default:  false,
+					},
+
+					"health_monitoring_enabled": {
+						Type:     pluginsdk.TypeBool,
+						Optional: true,
+						ForceNew: true,
+						Default:  false,
+					},
+
+					"incident_logs_enabled": {
+						Type:     pluginsdk.TypeBool,
+						Optional: true,
+						ForceNew: true,
+						Default:  false,
+					},
+				},
+			},
+		},
+
+		"domain": {
+			Type: pluginsdk.TypeString,
+			// The O+C is specified here because the service will set a default value to the hostname if not specified.
+			Optional: true,
+			Computed: true,
+			ForceNew: true,
+		},
+
+		"grid_image_ocid": {
+			Type:         pluginsdk.TypeString,
+			Optional:     true,
+			ForceNew:     true,
+			ValidateFunc: validation.StringIsNotEmpty,
+		},
+
+		"license_model": {
+			Type:         pluginsdk.TypeString,
+			Optional:     true,
+			ForceNew:     true,
+			ValidateFunc: validation.StringInSlice(exadbvmclusters.PossibleValuesForLicenseModel(), false),
+		},
+
+		"network_security_group_cidr": {
+			Type: pluginsdk.TypeList,
+			// The O+C is specified here because the service will set a default value to the network_security_group_cidr if not specified.
+			Optional: true,
+			Computed: true,
+			ForceNew: true,
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"destination_port_range": {
+						Type:     pluginsdk.TypeList,
+						Required: true,
+						MinItems: 1,
+						Elem: &pluginsdk.Resource{
+							Schema: map[string]*pluginsdk.Schema{
+								"max": {
```

Comment:
please use full words
```suggestion
								"maximum": {
```